### PR TITLE
New, simpler pareto front optimization

### PR DESCRIPTION
**Pull request by AlCap23**
#130

### CODE REVIEW

**Comment by AlCap23** [comment]
This seems wrong. Locally, the model is recovered. The docs do not recover the model, and tests recover a third one.

**Comment by AlCap23** [comment]
For reference

https://gist.github.com/ArnoStrouwen/22159e4c30196dc76671f84fc57eb687

**Comment by ChrisRackauckas** [comment]
Interesting. What are the sources of randomness here? If you set a random seed does it always give the same result, or is it something deeper?

**Comment by AlCap23** [comment]
AFAIK there is none. That's what's bugging me the most. But I'll try that on Sunday / Monday.

Did you try to run it on your machine by any chance? 

I will give this a shot on my old laptop as well, just in case. Even if I think the hardware should not be the issue.

**Comment by AlCap23** [comment]
I don't know if this is a win, but after changing to Ubuntu 20.04 and Julia 1.5 the test is failing for me as well :).

**Comment by ChrisRackauckas** [comment]
Looks like it's good now?

**Comment by AlCap23** [comment]
Nope. Michaelis is still wrong by a small factor ( and again not on my laptop but just on travis ). See the log of the tests for that.

But in general, yes. This seems to work and is a simple enough way while being flexible.

**Comment by AlCap23** [comment]
Works! Woohooo!

**Comment by ChrisRackauckas** [comment]
what was the trick?

**Comment by AlCap23** [comment]
Honestly, I do not know yet. I am working towards that.

I added the max convergence to `ISindy` and `ADM`, which might help. I do not think that `view` should have anything to do with that.

File: src/sindy/isindy.jl [hunk 1]
```diff
@@ -30,35 +30,35 @@ function ISInDy(X::AbstractArray, Ẋ::AbstractArray, Ψ::Basis; alg::Optimize.A
...
 
-    opt_front = ParetoFront(1, scalarization = alg)
-    tmp_front = ParetoFront(1, scalarization = alg)
 
+    fg(xi, theta) = (g∘f)(xi, theta)
+
+    iters = 0
 
-    @inbounds for i in 1:size(Ẋ, 1)
+    @simd for i in 1:size(Ẋ, 1)
```
Comment (ChrisRackauckas) [inline src/sindy/isindy.jl:38]: this isn't a simd-able loop. The macro will just be a no-op

Comment (AlCap23) [reply src/sindy/isindy.jl:38]: Alright. I will change that.

**Pull request merged**